crates/chainio/clients/avsregistry/src/writer.rs

```diff
@@ -849,4 +851,158 @@ mod tests {
             .unwrap();
         assert!(!is_registered);
     }
+
+    #[tokio::test]
+    async fn test_avs_reader_methods() {
```

Comment:
Can we move this to `reader.rs`?

Reply:
then wewould have to move the avs writer test functions also to reader , the `build_avs_registry_chain_writer`, `test_register_operator`

Reply:
We can move them to the crate's root, like we did with `elcontracts`:

https://github.com/Layr-Labs/eigensdk-rs/blob/f717f43fdd8f5201f7cce2606d9dab39c6c988ee/crates/chainio/clients/elcontracts/src/lib.rs#L14

Reply:
👍